cmd/observe/observe_test.go

```diff
@@ -19,11 +19,40 @@ import (
 	"time"
 
 	"github.com/cilium/cilium/api/v1/observer"
+	monitorAPI "github.com/cilium/cilium/pkg/monitor/api"
 	"github.com/cilium/hubble/pkg/defaults"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+func TestEventTypes(t *testing.T) {
+	// Make sure to keep event type slices in sync. Agent events, debug
+	// events and recorder captures have separate subcommands and are not
+	// supported in observe, thus the -3. See eventTypes godoc for details.
+	require.Len(t, eventTypes, len(monitorAPI.MessageTypeNames)-3)
+	for _, v := range eventTypes {
+		require.Contains(t, monitorAPI.MessageTypeNames, v)
+	}
+	eventTypesContainsValue := func(et string) bool {
+		for _, v := range eventTypes {
+			if v == et {
+				return true
+			}
+		}
+		return false
+	}
+	for k := range monitorAPI.MessageTypeNames {
+		switch k {
+		case monitorAPI.MessageTypeNameAgent,
+			monitorAPI.MessageTypeNameDebug,
+			monitorAPI.MessageTypeNameRecCapture:
+			continue
+		}
+		require.True(t, eventTypesContainsValue(k))
```

Comment:
Would [`require.Contains`](https://pkg.go.dev/github.com/stretchr/testify/require#Contains) be useful here?

Reply:
Thanks for the hint, changed to use `require.Contains`.